server/setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import setup
+
+
+setup(
+    name="kadalu-quotad",
+    version="0.1.0",
+    packages=["kadalu-quotad"],
```

Comment:
`-` will cause problem for package name. While creating console scripts it imports as `import kadalu-quotad` that may be problem. Change package name and dir name to `kadalu_quotad`